lua/lspinstall/servers/java.lua

```diff
@@ -0,0 +1,80 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {
+  "./jdtls.sh",
+  vim.fn.stdpath('data') .. '/lspinstall/java/workspace/' ..
+  string.gsub(
+    vim.fn.fnamemodify(vim.fn.getcwd(), ':p'),
+    "/",
+    "-"
+  )
+}
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
```

Comment:
Can this be run multiple times without errors? This is so we can update the server by running `:LspInstall java` again.